pkg/server/request.go

```diff
@@ -69,32 +66,37 @@ func SendRequest(request *http.Request, timeout int) (*http.Response, error) {
 }
 
 // ProcessRequest - Builds and sends file requests to the API
-func ProcessRequest(endpoint string, uploadOptions map[string]string, fileFieldData map[string]string, timeout int) error {
-
-	log.Info("Uploading to " + endpoint)
-
+func ProcessRequest(endpoint string, uploadOptions map[string]string, fileFieldData map[string]string, timeout int, fileName string, dryRun bool) error {
 	req, err := BuildFileRequest(endpoint, uploadOptions, fileFieldData)
 
 	if err != nil {
 		return fmt.Errorf("error building file request: %w", err)
 	}
 
-	res, err := SendRequest(req, timeout)
+	if !dryRun {
+		log.Info("Uploading " + filepath.Base(fileName) + " to " + endpoint)
 
-	if err != nil {
-		return fmt.Errorf("error sending file request: %w", err)
-	}
+		res, err := SendRequest(req, timeout)
 
-	b, err := io.ReadAll(res.Body)
+		if err != nil {
+			return fmt.Errorf("error sending file request: %w", err)
+		}
 
-	if err != nil {
-		return fmt.Errorf("error reading body from response: %w", err)
-	}
+		b, err := io.ReadAll(res.Body)
+
+		if err != nil {
+			return fmt.Errorf("error reading body from response: %w", err)
+		}
 
-	statusOK := res.StatusCode >= 200 && res.StatusCode < 300
+		statusOK := res.StatusCode >= 200 && res.StatusCode < 300
+
+		if !statusOK {
+			return fmt.Errorf("%s : %s", res.Status, string(b))
+		}
 
-	if !statusOK {
-		return fmt.Errorf("%s : %s", res.Status, string(b))
+		log.Success("Uploaded " + filepath.Base(fileName))
+	} else {
+		log.Success("(dryrun) Uploaded " + filepath.Base(fileName))
```

Comment:
I would still be nervous about this wording - "Uploaded" still sounds like something happened.

I would personally prefer something that made it absolutely clear nothing had happened. Something like:

```go
log.Success("(dryrun) Skipping upload of " + filepath.Base(fileName) + " to " + endpoint)
```

Reply:
Thoughts @tomlongridge ?

Reply:
Agreed - maybe also `.Info` log to underscore it?

Are you going to do equivalent for `Create.go`?